ivadomed/object_detection/utils.py

```diff
@@ -239,12 +239,12 @@ def load_bounding_boxes(object_detection_params, subject_path_list, slice_axis,
     bounding_box_dict = {}
     if object_detection_params is None or object_detection_params['object_detection_path'] is None:
         return bounding_box_dict
-    bounding_box_path = os.path.join(object_detection_params['path_output'], 'bounding_boxes.json')
-    if os.path.exists(bounding_box_path):
-        with open(bounding_box_path, 'r') as fp:
+    bounding_box_path = Path(object_detection_params['path_output'], 'bounding_boxes.json')
```

Comment:
For all dictionary keys, direct referencing dict key string like this WILL throw `KeyNotFound exception` if the dict does not contain it whereas using `example_dic.get("keyname", None)` type pattern will negate this issue and auto return None (or what else you specify there, if not specified, it will be default "None"). 

This is another anti-pattern to avoid. 
https://deepsource.io/blog/8-new-python-antipatterns/#5-not-using-get-to-return-default-values-from-a-dictionary

Whenever you can, and also in the future for the `Keyword.py` Phase II refactoring, try to watch out for this.